prov/efa/src/rxr/rxr_pkt_type_req.c

```diff
@@ -965,6 +965,17 @@ void rxr_pkt_handle_dc_longcts_rtm_send_completion(struct rxr_ep *ep,
 		rxr_cq_handle_tx_completion(ep, tx_entry);
 }
 
+void rxr_pkt_handle_runtread_rtm_send_completion(struct rxr_ep *ep,
+						 struct rxr_pkt_entry *pkt_entry)
+{
+	struct rxr_tx_entry *tx_entry;
+
+	tx_entry = (struct rxr_tx_entry *)pkt_entry->x_entry;
+	tx_entry->bytes_acked += rxr_pkt_req_data_size(pkt_entry);
+	if (tx_entry->total_len == tx_entry->bytes_acked)
```

Comment:
is the total_len here including the size of data to be read or just the data to be runt?

Reply:
Yes. It include both send and read. Note that `bytes_acked` was updated in function `rxr_pkt_handle_eor_recv` for the read part too.